lib/managers/webhooks.js

```diff
@@ -126,6 +126,19 @@ function Webhooks(client) {
 }
 
 /**
+ * Primary signature key to protect webhooks against attacks.
+ * @static
```

Comment:
These two JSDoc blocks should also have type information, e.g. `@type {?string}`